### PR TITLE
Replace subreddit shortening with css solition

### PR DESCRIPTION
**Pull request by danehansen**
:eyeglasses: @curioussavage

### CODE REVIEW

File: src/views/components/TopNav.jsx [hunk 1]
```diff
@@ -87,7 +81,7 @@ class TopNav extends React.Component {
     if (subredditName) {
       var breadcrumbLink = '/' + this.state.subredditName;
       var breadcrumbContents = subredditName;
-      currentSub = '?subreddit=' + subredditName;
+      currentSub = '?subreddit=' + this.state.subredditName;
```
Comment (curioussavage) [inline src/views/components/TopNav.jsx]: currentSub shouldn't have the `r/` prefix.

maybe we could just get the substring inline and axe `shorten` line 79 should be removed too.

**Comment by curioussavage** [comment]
:haircut:

**Comment by curioussavage** [comment]
uggh.. :nail_care:

**Comment by danehansen** [comment]
@curioussavage :haircut: Good catch.

**Comment by curioussavage** [comment]
:fish:

**Comment by curioussavage** [comment]
actually sorry one little nit. on `shorten` shouldn't take a second argument.

**Comment by curioussavage** [comment]
re-:fish:

**Pull request merged**